parameter_tools/src/main/java/coppercore/parameter_tools/path_provider/EnvironmentPathProvider.java

```diff
@@ -13,9 +13,19 @@ public EnvironmentPathProvider(Environment environment, String filepath, String
         this.filepath = filepath;
     }
 
+    /**
+     * Find path for file for the current enviroment. If File does not exist then it returns the
+     * desired location for the file to be made in.
+     */
+    @Override
+    public String resolveWritePath(String file) {
+        return resolvePath(file, environment.getDefaultToEnvironment());
+    }
+
+    /** Gets path to file This will throw an error if the file does not exist */
```

Comment:
this doesn't seem to throw an error if the file doesn't exist - let's update the comment or the code accordingly